tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -1897,12 +1897,29 @@ tensorflow::Status ConvertFusedBatchNorm(
     }
     if (scale_weights.type_ == tensorflow::DT_FLOAT) {
       for (size_t i = 0; i < nweight; ++i) {
-        float scale = (static_cast<float const*>(scale_weights.GetValues()))[i];
-        float offset =
-            (static_cast<float const*>(offset_weights.GetValues()))[i];
-        float mean = (static_cast<float const*>(mean_weights.GetValues()))[i];
-        float variance =
-            (static_cast<float const*>(variance_weights.GetValues()))[i];
+        float scale, offset, mean, variance;
```

Comment:
Can we use a for loop here also? Everything is from input.at(i).weights().

Reply:
will do.